spec/quality/documentation_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../spec_helper'
+require 'kramdown'
+
+RSpec.describe 'Documentation' do
+  files = Dir.glob(File.expand_path('../../**/*.md', __dir__))
+  files.each do |file|
+    describe "from #{file}" do
+      text = File.read(file)
+      doc = Kramdown::Document.new(text, input: 'GFM')
+      blocks = doc.root.children
+      blocks.each do |block|
+        next unless [:codeblock, :codespan].include?(block.type)
+        next unless block.attr['class'] == 'language-ruby'
```

Comment:
This means that blocks annotated with 'Ruby' are skipped. Ideally, I want to test those too, but it breaks for the README.md because one of the examples loads up `pry`.